### ISSUE TITLE
[Bug Report][3.0.0-alpha.11] v-menu and v-tooltip doesn't work : Vue warn on v-on="on"

### ISSUE THREAD

**Issue by antodasil** [vuetifyjs/vuetify#14299]
### Environment
**Vuetify Version:** 3.0.0-alpha.11
**Vue Version:** 3.0.0
**Browsers:** Firefox 93.0
**OS:** Windows 10

### Steps to reproduce
Add a v-menu with a <template #activator="{on, attrs}"> and a \<v-button v-on="on" v-bind="attrs"> within, the menu doesn't show when button is clicked.

Same problem for v-tooltip

### Expected Behavior
A menu is shown when button is click

### Actual Behavior
Click on button does nothing.

A v-warn is shown in console : 
```
[Vue warn]: v-on with no argument expects an object value. 
  at <VOverlay modelValue=false onUpdate:modelValue=fn<onUpdate:modelValue> class="v-menu"  ... > 
  at <VMenu offset-y="" > 
  at <VApp> 
  at <App>
```

### Reproduction Link
<a href="https://codepen.io/anto_dasil/pen/YzxWjYE" target="_blank">https://codepen.io/anto_dasil/pen/YzxWjYE</a>

### Other comments
I tried to log on and attrs, they both are undefined

**Comment by antodasil** [comment]
Oh This is why on and attrs was undefined. Thank you for your help and all the great job you do on Vuetify

**Comment by magyarb** [comment]
@KaelWD Thanks, this makes it work, however using v-menu still causes a hydration failure in nuxt.

The error:

```
[Vue warn]: Hydration children mismatch in <div>: server rendered element contains fewer child nodes than client vdom. 
  at <VOverlay ref=Ref< undefined > class="v-menu" attach=false  ... > 
  at <VMenu offset-y="" > 
  at <VToolbarItems> 
```


The reproduction:
[https://stackblitz.com/edit/node-htkewu?file=app.vue](https://stackblitz.com/edit/node-htkewu?file=app.vue)

Current workaround is surrounding v-menus with a `<client-only>` tag, but that is not a nice solution.

**Comment by KaelWD** [comment]
I just fixed that: #15323

**Comment by magyarb** [comment]
Thanks for fixing the problem! In the demo, it looks like it is solved. But if you use layouts and pages in nuxt3, the same problem appears again (and duplicates the buttons next to the v-menu in the toolbar).

Here is the modified reproduction link:
https://stackblitz.com/edit/node-htkewu
